src/api/app/models/event_subscription.rb

```diff
@@ -26,6 +26,7 @@ class EventSubscription < ApplicationRecord
   belongs_to :user, inverse_of: :event_subscriptions
   belongs_to :group, inverse_of: :event_subscriptions
   belongs_to :token, inverse_of: :event_subscriptions
+  belongs_to :package
```

Comment:
@danidoni this line is already in master, I would drop the commit that introduces this from your PR and rebase once more against master

Reply:
exactly, I had to add it for the specs to work.

Reply:
Okay